src/sw/redis++/queued_redis.hpp

```diff
@@ -78,9 +78,9 @@ auto QueuedRedis<Impl>::command(Cmd cmd, Args &&...args)
 template <typename Impl>
 template <typename ...Args>
 QueuedRedis<Impl>& QueuedRedis<Impl>::command(const StringView &cmd_name, Args &&...args) {
-    auto cmd = [](Connection &connection, const StringView &cmd_name, Args &&...args) {
+    auto cmd = [](Connection &connection, const StringView &lcmd_name, Args &&...largs) {
```

Comment:
Rename `lcmd_name` as `name`, and rename `largs` as `params`. Please also modify other `lcmd_name` and `largs` variables.